src/drivers/amd/Kconfig

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config AMD_BINARY_BUILD
+	bool
+	default n
+	help
+	  Select this if the platform need firmware binaries.
```

Comment:
Sorry, not sure I understand the comment correctly. What's the alternative? What does `CONFIG_AMD_BINARY_BUILD=n` mean? No firmware binaries needed? But SOF **is** providing such a firmware "binary?" Or do you mean **additional** binary-only firmware parts?

Reply:
We meant to include the "xsram_window" in the data section using this config. By default xsram_window is being present in .fw_metadata section in rimage. But In some cases, we need this data to be present in data section of rimage.

Reply:
@saisurya-ch maybe you could add a bit of information here in an incremental PR, if I were building SOF for AMD, I wouldn't be sure whether I need this option